xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -16,6 +16,7 @@
 #include "utils/Archive.h"
 #include "utils/StringUtils.h"
 #include "utils/Variant.h"
+#include "utils/log.h"
```

Comment:
Needed?

Reply:
Nope.  And yeah, the Estuary changes were only small although the initial commit included some other files that I had edited whilst trying to get it to display what I wanted.